release_scripts/branches.py

```diff
@@ -43,7 +43,7 @@ def write_instetc_version(version: str):
     logging.info(f"Writing version '{version}' to '{INSTETC_TEMPLATE_ABSOLUTE_PATH}'.")
 
     with open(INSTETC_TEMPLATE_ABSOLUTE_PATH, "r") as file:
-        data = re.sub(r"0\.0\.0(\.\$WCREV\$)", rf"{version}\1", file.read(), count=1)
+        data = re.sub(r"(field\(VAL, \").*(\.\$WCREV\$\"\))", rf"\g<1>{version}\g<2>", file.read(), count=1)
 
```

Comment:
If we fail to match the pattern it should be an error, but currently it would just return the original string. Could we check we actually have a match for the pattern as well as doing a replace?